apiserver/plane/app/views/issue/base.py

```
@@ -1014,6 +1014,26 @@ def get(self, request, slug, project_id):
                 .values("count")
             )
         )
+
+        # check for the project member role, if the role is 5 then check for the guest_view_all_features
```

Ruff (0.11.9)

Line too long (104 > 88)

(E501)